cogs/uptime.py

```diff
@@ -4,52 +4,21 @@
 from discord import app_commands
 from discord import Interaction
 from discord import TextChannel
```

Comment:
`TextChannel` is probably not used here either 🤔 Hopefully when #12 lands we won't have to bother about this haha.